### ISSUE TITLE
Vuex is not installed when the user selects "Install vuex? (state management) Yes"

### ISSUE THREAD

**Issue by msaelices** [nativescript-vue/vue-cli-template#118]
### Version
N/A

### Reproduction link
[https://nativescript-vue.org/en/docs/getting-started/quick-start/](https://nativescript-vue.org/en/docs/getting-started/quick-start/)
### Platform and OS info
All platforms

### Steps to reproduce
vue init nativescript-vue/vue-cli-template testapp

Select "Yes" in the "Install vuex? (state management)" question.

### What is expected?
Install and use vuex

### What is actually happening?
The app is exactly the same as if we answer no to the vuex question.

**Comment by ghost** [comment]
I usually do this manually:

`$ mkdir ./app/store`
`$ touch ./app/store/index.js`

Then in the created file you can start with the following code.

```js
import Vue from 'nativescript-vue'
import Vuex from 'vuex'

Vue.use(Vuex)

const store = new Vuex.Store({
  //
})

Vue.prototype.$store = store;

export default store;
```

Finally you can do this in your app-main.js.

```js
import Vue from 'nativescript-vue'
import Store from '@/store'

// Main component
import Main from '@/components/Main'

// Prints Vue logs when --env.production is *NOT* set while building
Vue.config.silent = (TNS_ENV === 'production')

const vm = new Vue({
  render: h => h('frame', [h(Main)]),
  store: Store
}).$start()
```

**Comment by rahimnathwani** [comment]
@msaelices it looks like you fixed this issue in #124 

Is that right, or is @gregorip02's suggestion preferred (making a `store` folder instead of a single file `store.js`)?

**Comment by msaelices** [comment]
@rahimnathwani yes I did it.
